to.etc.domui/src/main/java/to/etc/domui/component/misc/MsgBox.java

```diff
@@ -287,6 +287,16 @@ public static void info(NodeBase dad, NodeContainer string) {
 		message(dad, Type.INFO, string);
 	}
 
+	/**
+	 * Display some information; Call the onAnswer handler when 'Ok' is clicked.
+	 * @param dad
+	 * @param string
+	 * @param onAnswer
+	 */
+	public static void info(NodeBase dad, String string, final IAnswer onAnswer) {
```

Comment:
We no longer use final in those parameters. It makes the already very verbose Java code even more verbose, and does not really add value. I personally have no problem with people changing parameter values anyway.

Reply:
Btw, perhaps consider using MsgBox2?
```
MsgBox2.on(this).info().onanswer(....).
```